### ISSUE TITLE
MJWSD05MMC wrong formattred 30 date character 

### ISSUE THREAD

**Issue by ariekraakjr** [pvvx/ATC_MiThermometer#401]
I love this firmware for the my bt devices! (have ten of them around the house) 
My MJWSD05MMC(the rest are the small ones) is formatting the 3 in 30 wrong at the date place(see pic)
Is there a solution for this?
Use the 4.4  software.

![IMG_20231030_092319](https://github.com/pvvx/ATC_MiThermometer/assets/36099467/ff2b4d8f-7186-4718-b89a-d707ada9f30e)

**Comment by pvvx** [comment]
Unfortunately, the segments of the highest digit are combined hardware on the LCD display.

```
      --6.0--         --6.4--               --7.4--         --8.4--
    |         |     |         |           |         |     |         |
   5.4       5.5   5.1       6.5      /  6.1       7.5   7.1       8.5
    |         |     |         |      /    |         |     |         |
      --5.6--         --5.2--      5.0      --6.2--         --7.2--
    |         |     |         |    /      |         |     |         |
   6.0       5.7   5.3       6.6  /      6.3       7.6   7.3       8.6
    |         |     |         |           |         |     |         |
      --6.0--         --6.7--               --7.7--         --8.7--
```
we'll have to swap the "month" and "days".
This will be fixed in version 4.5  (in an update, in an hour).

**Comment by ariekraakjr** [comment]
Wow, that's fast. Thanks
